src: teach llnode how to print Symbol properties names

mmarchini opened the pull request:
Still need to add the metadata to V8 before landing this.

Closes: https://github.com/nodejs/llnode/issues/156

cjihrig commented on src/llv8.cc:
```diff
@@ -955,6 +961,15 @@ std::string HeapNumber::Inspect(Error& err) {
 
 
+std::string Symbol::ToString(Error& err) {
+  if (!String::IsString(v8(), Name(err), err)) {
```
Any reason not to do `Symbol()`/`Symbol('foo')`, like would be shown in JS?

mmarchini replied:
You mean for unnamed symbols? `Symbol()` sounds like a good idea in this case.

mmarchini commented:
Rebased, changed display format to `Symbol()/Symbol('foo')` and added code to fallback to `<non-string>` when Symbols metadata is not available (Node.js 6 and 8).

mmarchini commented:
Landed in e30beba
